pkg/printers/internalversion/describe_test.go

```diff
@@ -1682,18 +1682,12 @@ func TestDescribeServiceAccount(t *testing.T) {
 	if err != nil {
 		t.Errorf("unexpected error: %v", err)
 	}
-	expectedOut := `Name:                bar
-Namespace:           foo
-Labels:              <none>
-Annotations:         <none>
-Image pull secrets:  test-local-ref (not found)
-Mountable secrets:   test-objectref (not found)
-Tokens:              <none>
-Events:              <none>` + "\n"
-	if out != expectedOut {
-		t.Errorf("expected : %q\n but got output:\n %q", expectedOut, out)
+	expectedOut := []string{"bar", "foo", "Image pull secrets", "test-local-ref (not found)", "Mountable secrets:", "test-objectref (not found)"}
+	for _, expected := range expectedOut {
```

Comment:
Well, I don't like changing to this, since `expectedOut` is exactly what we expect to see.